CRFModuleValidation/MyJourneyViewController.swift

```diff
@@ -0,0 +1,781 @@
+            let overallWidth = self.preferredSizeView.bounds.size.width
+            headerCell.detailWidth.constant = min(overallWidth - 48, 290)
+            headerCell.titleToDetail.constant = self.view.bounds.size.width < 375 ? 12 : 24
+        }
+        else if let dateCell = cell as? MyJourneyDateCell, let scheduleSection = self.scheduleSection(at: indexPath) {
+            if indexPath.section == self.todaySectionIndex {
+                dateCell.dateLabel.text = Localization.localizedString("Today")
+            }
+            else {
+                let dateFormatter = DateFormatter()
+                dateFormatter.setLocalizedDateFormatFromTemplate("MMMMd")
+                dateCell.dateLabel.text = dateFormatter.string(from: scheduleSection.date)
+            }
+            dateCell.adjustInsets()
+        }
+        
+        return cell
+    }
+    
+    func tableView(_ tableView: UITableView, heightForRowAt indexPath: IndexPath) -> CGFloat {
+        
+        switch(cellIdentifier(at: indexPath)) {
+        case .todayHeader:
+            return UITableViewAutomaticDimension
+        
+        case .date:
+            return 36
+            
+        case .scheduleSection:
+            return MyJourneyActivityViewCell.preferredSize(isFullSize: usesFullSize(for: indexPath.section)).height
+            
+        case .separator:
+            if shouldShowFirstDayOnly && usesFullSize(for: indexPath.section + 1) {
+                return self.preferredSizeView.bounds.size.height - tableView.tableHeaderView!.bounds.size.height - 160
+            }
+            else if usesFullSize(for: indexPath.section) ||
+                usesFullSize(for: indexPath.section + 1) ||
+                (indexPath.section == scheduleSections.count - 1) {
+                return 42   // last section and surrounding today section are bigger
+            }
+            else {
+                return 24
+            }
+        }
+    }
+    
+    func tableView(_ tableView: UITableView, estimatedHeightForRowAt indexPath: IndexPath) -> CGFloat {
+        return 160
+    }
+    
+    
+    // MARK: show the schedule item
+    
+    func taskGroupForSegue(withIdentifier identifier: String, sender: Any?) -> (TaskGroup, Date)? {
+        
+        // Is this the add Activity segue?
+        if identifier == addActivitySegue {
+            return (TaskGroup.addActivities, Date())
+        }
+        
+        // Is this a task group segue?
+        if identifier.hasPrefix(taskGroupViewControllerSegue),
+            let button = sender as? UIButton {
+            
+            let isYesterdayCell = button.tag >= MyJourneyActivityViewCell.isYesterdayButtonTagFactor
+            let adjustSectionIndex = isYesterdayCell ? (todaySectionIndex + 1) : todaySectionIndex
+            
+            if let scheduleSection = self.scheduleSection(at: IndexPath(item: 0, section: adjustSectionIndex)),
+                isYesterdayCell || button.tag < scheduleSection.items.count {
```

Comment:
You didn't comment out `isYesterdayCell` here.

Reply:
thanks